internal/k8sutils/utils.go

```diff
@@ -141,3 +141,20 @@ func GetExpectedTPUNodePoolSize(node *corev1.Node) (int32, error) {
 
 	return int32(product / acceleratorCount), nil
 }
+
+func TpuTopologyToChipCount(topo string) (int, error) {
```

Comment:
Prefer to refactor the funcs to reuse the topo product logic already present in func above

Reply:
fixed in b767f78